### PR TITLE
PR: Product page & Landing Page

### PR DESCRIPTION
**Pull request by pavan6115**
- Done with Landing Page of Ecom
- Added Product Page for Ecom

cremech-product-landing

### CODE REVIEW

File: pages/product-page/productPage.css [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+.visible {
+    color: black;
+    font-size: 1rem;
+    font-weight: 700;
+}
+
+.filter {
+    grid-area: sidebar;
+    position: fixed;
+    top: 49px;
```
Comment (Nishant-96) [inline pages/product-page/productPage.css:92]: Try to use relative units.

**Review by Nishant-96 — APPROVED** [review]
Looks good to merge.

**Review by xsaurabhsuthar — APPROVED** [review]

File: pages/product-page/productPage.css (hunk 1 above)
Comment (pavan6115) [reply pages/product-page/productPage.css:92]: ok

**Review by Tejs1 — APPROVED** [review]
lgtm

**Pull request merged**